geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert2/SimpleFeatureConverterFactory.scala

```diff
@@ -31,5 +32,26 @@ trait SimpleFeatureConverterFactory {
     * @param sft simple feature type, if known ahead of time
     * @return
     */
-  def infer(is: InputStream, sft: Option[SimpleFeatureType] = None): Option[(SimpleFeatureType, Config)] = None
+  @deprecated("Use infer(InputStream, Option[String], Option[SimpleFeatureType]")
```

Comment:
Do you mean:
  @deprecated("Use infer(InputStream, Option[SimpleFeatureType],  Option[String]")
?

Reply:
yes, thanks